GVRf/Framework/src/org/gearvrf/GVRActivity.java

```diff
@@ -319,12 +319,17 @@ boolean updateSensoredScene() {
      * @param view Is a {@link GVRView} that draw itself into some
      *            {@link GVRViewSceneObject}.
      */
-    public void registerView(View view) {
+    public void registerView(final View view) {
         /* The full screen should be updated
            otherwise just the children's bounds may be refreshed. */
         mRenderableViewGroup.setClipChildren(false);
```

Comment:
It needs to go into the runnable. It crashes when called from the gl thread:

````java
             System.err  W  android.view.ViewRootImpl$CalledFromWrongThreadException: Only the original thread that created a view hierarchy can touch its views.
                         W      at android.view.ViewRootImpl.checkThread(ViewRootImpl.java:7648)
                         W      at android.view.ViewRootImpl.invalidateChildInParent(ViewRootImpl.java:1107)
                         W      at android.view.ViewRootImpl.invalidateChild(ViewRootImpl.java:1102)
                         W      at android.view.View.invalidateInternal(View.java:12959)
                         W      at android.view.View.invalidate(View.java:12923)
                         W      at android.view.ViewGroup.setClipChildren(ViewGroup.java:3743)
                         W      at org.gearvrf.GVRActivity.registerView(GVRActivity.java:325)
                         W      at org.gearvrf.scene_objects.view.GVRTextView.<init>(GVRTextView.java:33)
                         W      at org.gearvrf.scene_objects.view.GVRTextView.<init>(GVRTextView.java:61)
                         W      at org.gearvrf.scene_objects.view.GVRTextView.<init>(GVRTextView.java:46)

````